lang/java/java.talon

```diff
@@ -30,8 +34,8 @@ type {user.code_type} array:
     insert(user.code_type)
     user.code_operator_subscript()
 
-[state] {user.java_access_modifiers}: 
-    insert(user.java_access_modifiers)
+[state] {user.java_access_modifiers}|{user.java_keywords}: 
```

Comment:
I  think you need to put parentheses here, `[state] ({user.java_access_modifiers} | {user.java_keywords})`. Otherwise "state throw" will not work.

Reply:
Also, @knausj85 - any opinion on whether it's okay to have the java keywords bound without a preceding "state"? It looks like we don't do this for other languages, but we have done it in java.

Reply:
Oops. I prefer to require state for consistency/avoiding issues with short words, but looks like something I missed previously. I’d vote to leave as-is for this PR and maybe open an issue.

Reply:
This still needs parentheses.